src/metadata.rs

```diff
@@ -275,11 +275,42 @@ pub struct CredentialMetadataDisplayLogo {
     alt_text: Option<String>,
 }
 
-#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct AdditionalOAuthMetadata {
     #[serde(rename = "pre-authorized_grant_anonymous_access_supported")]
     pre_authorized_grant_anonymous_access_supported: Option<bool>,
+    pushed_authorization_request_endpoint: Option<AuthUrl>,
+    require_pushed_authorization_requests: Option<bool>,
+}
+
+impl Default for AdditionalOAuthMetadata {
+    fn default() -> Self {
+        AdditionalOAuthMetadata {
+            pre_authorized_grant_anonymous_access_supported: Some(false),
+            pushed_authorization_request_endpoint: None,
+            require_pushed_authorization_requests: Some(false),
+        }
+    }
 }
```

Comment:
No need to remove the `#[derive(Default)]` IMO, since omitted (`None`) means `false`.